### PR TITLE
Feature/css lint

### PR DESCRIPTION
**Pull request by ericawright**
reviewer: @bwinton 

### Changes
- add stylelint npm package
- add custom .stylelintrc
- lint all css files

### Notes
- not alphabetized due to cascading properties of CSS
- grouped positioning and grouped box-model stuff at the top, rest at bottom
- run eslint and stylelint with `npm run lint`
- stylelint will only run if there are **no** eslint errors

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        "border-right",
+        "border-bottom",
+        "border-left",
+        "border-radius",
+        "margin",
+        "margin-top",
+        "margin-right",
+        "margin-bottom",
+        "margin-left"
+      ]}], { "unspecified": "bottom" }],
```
Comment (bwinton) [inline .stylelintrc.json]: I think I'ld prefer `bottomAlphabetical`, but that could be a followup patch…

File: skin/base.css [hunk 2]
```diff
@@ -81,7 +83,7 @@
 /* XXX if I was smarter I'd find a way to make a swoop on the right side of the bar, under the window buttons */
 /* Maybe not - this conveniently leaves a hit target for dragging. */
 /*#nav-bar {
-  margin-right: 100px;
+margin-right: 100px;
```
Comment (bwinton) [inline skin/base.css]: I think we want to revert this change…

File: .stylelintrc.json (hunk 1 above)
Comment (ericawright) [inline .stylelintrc.json]: initially thought so too, but: vendor prefixes and cascading, sometimes the order can't be changed...

Comment (bwinton) [inline .stylelintrc.json]: Hmm.  Can you give me an example line?

Comment (ericawright) [inline .stylelintrc.json]: screen shot 2016-05-13 at 3 41 46 pm
I concede, it is smarter than me and knows about prefixes...
also, there doesn't seem to be any cases where the more specific attr would come before the general one - so I will make it alphabetical

Comment (bwinton) [inline .stylelintrc.json]: Yay!  \o/  😀
